src/dom/isdomnode.js

```diff
@@ -7,14 +7,20 @@
  * @module utils/dom/isdomnode
  */
 
-import isNative from '../lib/lodash/isNative';
-
 /**
  * Checks if the object is a native DOM Node.
  *
  * @param {*} obj
  * @returns {Boolean}
  */
 export default function isDomNode( obj ) {
-	return !!( obj && isNative( obj.addEventListener ) );
+	if ( obj ) {
+		if ( obj.defaultView ) {
+			return obj instanceof obj.defaultView.Document;
```

Comment:
Is document a node?

Reply:
Yes

![image](https://user-images.githubusercontent.com/1099479/33209458-28028db8-d116-11e7-97e2-849a4427b29c.png)

Reply:
Sorry, I actually wanted to know whether this code is needed at all and whether this code is semantically correct. But now I see that both things are true.